build.gradle

```diff
@@ -24,12 +26,23 @@ buildscript {
       'jetbrainsAnnotations': 'org.jetbrains:annotations:16.0.3',
       'guava': 'com.google.guava:guava:27.0.1-jre',
   ]
+
+  repositories {
+    mavenCentral()
+    gradlePluginPortal()
+  }
+
+  dependencies {
+    classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.7'
```

Comment:
no `ext.deps` on the new dependencies because they're only used once?

Reply:
yes